pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/relocation/RealtimeSegmentRelocator.java

```diff
@@ -72,40 +69,33 @@ protected void processTable(String tableNameWithType) {
         .hasHighLevelConsumerType()) {
       return;
     }
+    if (!InstanceAssignmentConfigUtils.shouldRelocateCompletedSegments(tableConfig)) {
```

Comment:
May be add a log saying the relocation is NO-OP?

Reply:
Added